### DISCUSSION TITLE
mac os How to solve this problem

### DISCUSSION THREAD

**Discussion by dorameonsu** [openai/whisper#441] [Q&A]
![image](https://user-images.githubusercontent.com/85793026/198913539-7685d0eb-16d2-41e5-844a-6cd2e18cd43d.jpeg)

**Comment by gmat** [comment]
Go with `pip install certifi` to get the Mozilla's CA Bundle. See https://pypi.org/project/certifi/

**Reply by TheHypnoo** [reply]
@gmat I have installed the certifi dependency, but it still does not work. It shows me the same error.

**Reply by TheHypnoo** [reply]
I just used this and it worked for me. I'm passing it on in case it happens to anyone else.
https://stackoverflow.com/questions/52805115/certificate-verify-failed-unable-to-get-local-issuer-certificate

**Reply by jsjoeio** [reply]
I had to restart my shell and then it started working.
